post_processing/plot/plot_helpers.jl

```diff
@@ -415,6 +418,17 @@ function generate_paperplots_held_suarez(fig_dir, nc_files; moist)
             if moist
                 global qt = nc["qt"][:]
             end
+            if haskey(nc, "sfc_flux_energy")
+                sfc_flux_energy = nc["sfc_flux_energy"][:]
+            end
+            if haskey(nc, "sfc_evaporation")
+                sfc_evaporation = nc["sfc_evaporation"][:]
+            end
+            if haskey(nc, "lw_flux_up")
+                # NOTE: diff = up - down
+                lw_flux_diff = nc["lw_flux_up"][:] - nc["lw_flux_down"][:]
+                sw_flux_diff = nc["sw_flux_up"][:] - nc["sw_flux_down"][:]
```

Comment:
```suggestion
                sw_flux_net = nc["sw_flux_up"][:] - nc["sw_flux_down"][:]
```
(and similarly for other places where it is used)